src/bp-core/admin/settings/bp-admin-setting-xprofile.php

```diff
@@ -61,8 +61,10 @@ public function settings_save() {
 		 *
 		 * @since BuddyBoss 1.0.0
 		 */
-		$bp_nouveau_appearance                             = bp_get_option( 'bp_nouveau_appearance', array() );
-		$bp_nouveau_appearance['user_front_page']          = isset( $_POST['bp-enable-member-dashboard'] ) ? $_POST['bp-enable-member-dashboard'] : 0;
+		$bp_nouveau_appearance = bp_get_option( 'bp_nouveau_appearance', array() );
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.ValidatedSanitizedInput.MissingUnslash, WordPress.Security.NonceVerification.Missing
+		$bp_nouveau_appearance['user_front_page'] = isset( $_POST['bp-enable-member-dashboard'] ) ? $_POST['bp-enable-member-dashboard'] : 0;
```

Comment:
`$_POST['bp-enable-member-dashboard'] `should be sanitized

Reply:
Done